src/IIIFPresentation/API/Features/Manifest/ManifestService.cs

```diff
@@ -123,6 +118,13 @@ private async Task<PresUpdateResult> CreateInternal(WriteManifestRequest request
         var (parentErrors, parentCollection) = await TryGetParent(request, cancellationToken);
         if (parentErrors != null) return parentErrors;
 
+        // can't have both items and painted resources, so items takes precedence
```

Comment:
Do we need to have this behaviour controlled by an appsetting or feature flag, or is it so simple that it can be stripped out again easily? I guess the updated tests when fully implemented will make it obvious if it's still doing this.

Reply:
I've added a setting to control this